### ISSUE TITLE
pagination failure (Invalid pagination token given) on a function

### ISSUE THREAD

**Issue by AshwiniRaysecur** [aws-amplify/amplify-console#433]
Description: We have a user view which presents records from a DynamoDB table to the end user. The control causes records to be fetched 10 at a time and a 'next token' is retrieved for pagination to advance the listing when more than 10 records are available. The feature had been working as expected but recently has started throwing an error when the 'next' button is used.

----

Expected result: clicking the next button should advance the listing to the next 10 records (if more than 10 are available).

Actual result: An error is thrown as follows:
----  ERROR OBJECT ----

```
[
  {
    "path": [
      "listSecurSamples"
    ],
    "data": null,
    "errorType": "DynamoDB:UserIllegalArgumentException",
    "errorInfo": null,
    "locations": [
      {
        "line": 2,
        "column": 3,
        "sourceName": null
      }
    ],
    "message": "com.amazonaws.deepdish.common.pagination.InvalidPaginationTokenException: Invalid pagination token given."
  }
]
```

----
Screenshot of the function and obtained error

<img width="1537" alt="screenshot" src="https://user-images.githubusercontent.com/59932758/74377622-4aa3fd00-4db2-11ea-8a08-f5894f5cee84.png">

----
A similar unresolved issue is reported here and can be followed:
https://github.com/aws-amplify/amplify-cli/issues/3411

**Comment by swaminator** [comment]
@AshwiniRaysecur are you using the Amplify Console for hosting this app?

**Comment by AshwiniRaysecur** [comment]
@swaminator Yes, we are using the Amplify Console to host the app

**Comment by Joycehao19** [comment]
@AshwiniRaysecur  Hi, we cut a ticket in our customer oncall list, now is in pending of investigation.

**Comment by nick-secur** [comment]
Thanks we'll stand by and continue monitoring this ticket...

**Comment by nick-secur** [comment]
Please note this issue is the same as reported here: [aws-amplify/amplify-cli#3411](https://github.com/aws-amplify/amplify-cli/issues/3411)

And it has been fixed in aws amplify version 4.13.4 ...

**Comment by swaminator** [comment]
This isn't an issue with the Amplify Console and is instead an issue with the service. Closing since reported fixed.
